types/draco3d/index.d.ts

```diff
@@ -0,0 +1,197 @@
+    GetAttributeInt32ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
+    GetAttributeUInt8ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
+    GetAttributeUInt16ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
+    GetAttributeUInt32ForAllPoints: (mesh: Mesh, attribute: Attribute, array: DracoArray) => void;
+    GetEncodedGeometryType: (buffer: DecoderBuffer) => GeometryType;
+}
+
+export interface DecoderBuffer {
+    Init: (array: Int8Array, byteLength: number) => void;
+}
+
+export interface DracoArray {
+    GetValue: (index: number) => number;
+}
+
+// tslint:disable-next-line:no-empty-interface
+export interface DracoFloat32Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoInt8Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoInt16Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoInt32Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoUInt8Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoUInt16Array extends DracoArray {}
+// tslint:disable-next-line:no-empty-interface
+export interface DracoUInt32Array extends DracoArray {}
+
+export interface Status {
+    ok: () => boolean;
+}
+
+export interface Attribute {
+    num_components: () => number;
+}
+
+// tslint:disable-next-line:no-empty-interface
+export enum GeometryType {}
+
+// tslint:disable-next-line:no-empty-interface
+export enum DataType {}
+
+export interface Mesh {
+    ptr: number;
+    num_faces: () => number;
+    num_points: () => number;
+}
+
+export interface MeshBuilder {
+    AddFacesToMesh(mesh: Mesh, numFaces: number, faces: Uint16Array | Uint32Array): void;
+    AddUInt8Attribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+    AddInt8Attribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+    AddUInt16Attribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+    AddInt16Attribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+    AddUInt32Attribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+    AddFloatAttribute(
+        mesh: Mesh,
+        attribute: number,
+        count: number,
+        itemSize: number,
+        array: TypedArray
+    ): number;
+}
+
+export {};
```

Comment:
Do you need this?

Reply:
Yeah, the DTS linter complains about `type TypedArray`, at the top of the file, not being exported if this isn't included. I'm just using as an internal type throughout the file to avoid repeating that list of types.